src/components/Timeperiod.tsx

```diff
@@ -10,60 +10,57 @@ import { InlineLoader } from './Loader';
 import Tooltip from './Tooltip';
 
 type TTimeperiodItemProps = {
-    category: typeof Intervals[0];
+    category: (typeof Intervals)[0];
```

Comment:
Are brackets needed here? Please, ignore if not relevant

Reply:
without brackets TS will try to take the first element if `Intervals`. but it needs to take the first element of `typeof Intervals`. Is that right  @ahmadtaimoor-deriv  ? :)

Reply:
The one Kate mentioned typeof Intervals[0] works the same as with bracket. I removed it , thanks